frontend/src/v5/ui/routes/viewer/ticketFiltersSetter/ticketFiltersSetter.component.tsx

```diff
@@ -15,7 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
(originally reported in https://github.com/3drepo/3drepo.io/pull/5492)
Changing pin condition in Tabular view and open viewer, pins aren't showing properly.
Without clicking background to trigger the ticket updated alert.
![05-missingpin](https://github.com/user-attachments/assets/cc446471-25a9-4354-81d7-d6301ea99eaf)
Clicking background to trigger the ticket updated alert.
![05-missingpin2](https://github.com/user-attachments/assets/d01b7d67-0a41-4da7-b7d9-06acb5fc4792)